src/allotropy/parsers/lines_reader.py

```diff
@@ -160,19 +158,15 @@ def pop_as_series(self, sep: str = " ") -> Optional["pd.Series[str]"]:
 
     def lines_as_df(
         self,
-        *,
```

Comment:
What is the purpose of removing this?

Reply:
The idea is to allow the function to pass all the kwargs directly to pandas, instead of the need to add the parameters one by one.